### DISCUSSION TITLE
How do teams share responsibilities for  manual accessibility testing?

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5716] [Accessibility]
Happy New Year everyone! This month, let’s talk about manual accessibility testing for keyboard navigation, assistive technology, and zoom magnification. These tests are critical steps to ensure accessibility.

Ideally, we should share responsibility for ensuring 508 compliance across the whole team, but sometimes decisions or testing may fall on a single individual.  

- Have you ever done manual testing? 
- What skills could each team member use to help conduct manual testing?
- Which manual testing skills do you think are the most important in order to cross-train others?
- How can USWDS support you and your team with these efforts? 

We’d like to hear about your experience!

And we'll be talking more about manual accessibility testing on [January 18 at the USWDS monthly call](https://digital.gov/event/2024/01/18/uswds-monthly-call-january-2024/).

**Comment by sarah-sch** [comment]
This [guidance from section508.gov](https://www.section508.gov/manage/deia-guidance/) offers some great pointers about how accessibility work can be shared across the team. (See the “Guidance, Resources, and Technical Assistance” section about halfway down the page.)

**Comment by YashuMishra** [comment]
Hi @amycole501  based on your question i have some feedback according to my experience in Accessibility testing:
-**Have you ever done manual testing?**
Yes, I have performed manual as well as automated testing for both web and application upon Wcag 2.1/2.2 guidelines.
**What skills could each team member use to help conduct manual testing?**
- So basically for manual testing just working over keyboard or tabbing through controls won't work team members should have basic knowledge or ARIA practices , expected behavior of controls such as buttons, combo box , placeholder , drop down and also of some basic HTML.

**Which manual testing skills do you think are the most important in order to cross-train others?**
-The best training which can give to the team members is on keyboard, let them know about controls expected behavior, ARIA practices and individually testing over dummy URL and match the gaps of bug findings among each other
**How can USWDS support you and your team with these efforts?**
- I think I have no answer for this because I am accessibility tester from India and Referred WCAG guidelines

Thanks 
Yash

**Reply by amycole501** [reply]
Good practices, Yash, and I agree about learning ARIA and how and when to implement it. Keyboard testing is one step but testing using assistive technology (AT) is important as well. And component-specific behavior is key to understand because AT interacts with components via the keyboard in different ways. Thanks for contributing!

**Comment by ericwbailey** [comment]
Hi there, found this Discussion via the newsletter. 

> What skills could each team member use to help conduct manual testing?

If there's one thing I wish I could stress over on my end of things it's that testing in Desktop VoiceOver is not paint a good picture for what many of our users experience. We are mostly a desktop-centric experience, so it would be NVDA or JAWS. 

When I was working on government services (back in the private sector now), the idea was sort of the opposite, in that I was hoping to get more Android/feature phone evaluation done. I feel there's a lot of survivor bias present in the current status quo because of this lack of testing.

>Which manual testing skills do you think are the most important in order to cross-train others?

Removing the screen to remove the visual bias. Something that's technically accessible may still be unusable because of the way the content is separated from the context of the visual design.

**Reply by amycole501** [reply]
All excellent tips! Knowing how your users access your content is the first step in getting accurate test results for your project and using the correct technology for testing.

Visual design shouldn't be the primary way a user experience is created. Leaning on proper structure, labeling, and descriptions/instructions will help AT users and honestly, anyone.
